### PR TITLE
Add info regarding serialization for RQ launcher plugin

### PR DESCRIPTION
**Pull request by jan-matthis**
## Motivation

Add info regarding serialisation/third party dependencies for RQ launcher plugin as per discussion in Zulip

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: website/docs/plugins/rq_launcher.md [hunk 1]
```diff
@@ -97,4 +97,6 @@ $ python my_app.py --multirun task=1,2,3,4,5
 [HYDRA] Polling job statuses every 1.0 sec
 ```
 
+Note that any third party library dependencies need to be installed on all nodes. For serialization of jobs [`cloudpickle`](https://github.com/cloudpickle/cloudpickle) is used.
```
Comment (omry) [inline website/docs/plugins/rq_launcher.md:100]: Correct me if I am wrong, but it's a bit more than this:
the dependencies needs to be installed in the Python interpreter used to run the RQ worker, right?

**Comment by omry** [comment]
Are you rebased?

File: website/docs/plugins/rq_launcher.md (hunk 1 above)
Comment (jan-matthis) [reply website/docs/plugins/rq_launcher.md:100]: Yes, how would you phrase it instead?

Comment (jan-matthis) [reply website/docs/plugins/rq_launcher.md:99]: Perhaps: 
> Note that any dependencies need to be installed in the Python environment used to run the RQ worker.

Comment (omry) [reply website/docs/plugins/rq_launcher.md:99]: Sounds good.

Did you rebase?

Comment (jan-matthis) [reply website/docs/plugins/rq_launcher.md:99]: Yes, I did: 
> This branch is 2 commits ahead of facebookresearch:master.

Not sure why tests are failing. I'll change the sentence to the one above. Hopefully they'll pass next time.

**Review by omry — APPROVED** [review]
Thanks.

**Pull request merged**